contracts/child/ChildToken/ChildMintableERC721.sol

```diff
@@ -73,15 +91,41 @@ contract ChildMintableERC721 is
         _burn(tokenId);
     }
 
+    /**
+     * @notice called when user wants to withdraw multiple tokens back to root chain
+     * @dev Should burn user's tokens. This transaction will be verified when exiting on root chain
+     * @param tokenIds tokenId list to withdraw
+     */
+    function withdrawBatch(uint256[] calldata tokenIds) external {
+
+        uint256 length = tokenIds.length;
+        require(length <= BATCH_LIMIT, "ChildMintableERC721: EXCEEDS_BATCH_LIMIT");
+
+        // Iteratively burn ERC721 tokens, for performing
+        // batch withdraw
+        for (uint256 i; i < length; i++) {
+
+            uint256 tokenId = tokenIds[i];
+
+            require(_msgSender() == ownerOf(tokenId), string(abi.encodePacked("ChildMintableERC721: INVALID_TOKEN_OWNER ", tokenId)));
+            withdrawnTokens[tokenId] = true;
+            _burn(tokenId);
```

Comment:
Will `_burn` revert if `tokenId` is non-existent?

Reply:
Yes [it'll](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/fb95a8b34bb34d797f5b1dcf64a7be5cce7219c4/contracts/token/ERC721/ERC721.sol#L280)